proxy/usr.bin.securedrop-proxy

```diff
@@ -0,0 +1,12 @@
+abi <abi/3.0>,
+
+include <tunables/global>
+
+/usr/bin/securedrop-proxy {
+  include <abstractions/base>
+  include <abstractions/nameservice>
```

Comment:
I think this is fine for now, but also flags that reqwest is presumably doing DNS lookups that it doesn't need to be, since the onion service hostname is resolved by Tor.